spec/requests/users/sign_in_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'User sign-in', type: :request do
+  subject { response }
+
+  let!(:user) do
+    User.create!(
+      first_name: 'john',
+      last_name: 'doe',
+      email: 'foo@bar.com',
+      password: '123456'
+    )
+  end
+
+  context 'when user signs in' do
+    before do
+      post user_session_path \
```

Comment:
Instead of using `\`, lets use

```
post(
  user_session_path,
  'user[email]': user.email,
  'user[password]': user.password
)
```

for these cases ;)

Reply:
the spec failed with that syntax, this one worked:
```
      post user_session_path,
           params: {
             'user[email]': user.email,
             'user[password]': user.password
           }
```
(rubocop complained about not having the 'params')